boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -286,6 +290,22 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
             break;
         }
 
+        /* Extra (input) safety. */
+        if (Regs.x.ecx < 20 || BiosMemoryMapSize < Regs.x.ecx)
+        {
+            ERR("BIOS returned an invalid entry length! (would-be-PcBiosMapCount = %lu, Entry length = (20 <=) %lu (<= %lu))\n\n",
```

Comment:
A size_t should be printed with `%Iu` (capital eye), or casted to another printable type.

Reply:
Thanks for the reminder.
Done.